tests/tasks/verify_settings.yml

```diff
@@ -0,0 +1,110 @@
+- name: Verify password
+  when: __verify_mssql_password is defined
+  block:
+    - name: Ensure that the mssql-server service is started
+      service:
+        name: mssql-server
+        state: started
```

Comment:
under what conditions do you want to check the password when the server is not already running?

Reply:
Apparently, this is expected that if service is not running, the role will start it. So my concern is incorrect.